diff --git a/node_modules/@discordjs/voice/dist/index.d.ts b/node_modules/@discordjs/voice/dist/index.d.ts
index 699102a..fbc7429 100644
--- a/node_modules/@discordjs/voice/dist/index.d.ts
+++ b/node_modules/@discordjs/voice/dist/index.d.ts
@@ -1,8 +1,9 @@
 import { Readable, ReadableOptions } from 'node:stream';
-import prism from 'prism-media';
+import * as prism from 'prism-media';
 import { TypedEmitter } from 'tiny-typed-emitter';
 import { GatewayVoiceServerUpdateDispatchData, GatewayVoiceStateUpdateDispatchData } from 'discord-api-types/v9';
-import WebSocket, { MessageEvent } from 'ws';
+import { MessageEvent } from 'ws';
+import * as WebSocket from 'ws'
 
 declare type Awaited<T> = T | Promise<T>;
 
